server/WebAPI/DTO/MovieDTO.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using WebAPI.Models;
+
+namespace WebAPI.DTO
+{
+    public class MovieDTO
+    {
+        public int Id { get; set; }
+        public string Title { get; set; }
+        public int Rate { get; set; }
+        public int ReleaseYear { get; set; }
+        public string ImageSource { get; set; }
+
+        public ICollection<ActorDTO> ActorsList { get; set; }
```

Comment:
why do you need actor dto list just for one string?
Actually I sure that you need ICollection<MovieListActorDTO > Actors { get; set; }
where MovieListActorDTO { ActorId, FirstName, LastName }
create you folder for dto connected to movie namespace WebAPI.DTO.Movies